### PR TITLE
chore: Add trustbloc driver

### LINKED ISSUE #104: Add trustbloc driver
https://github.com/trustbloc/trustbloc-did-method/blob/master/docs/spec/trustbloc-did-method.md

### PR DESCRIPTION
**Pull request by fqutishat**
closes #104

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Comment by fqutishat** [comment]
@peacekeeper can you review this PR

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ See https://uniresolver.io/ for a publicly hosted instance of a Universal Resolv
 
 ## Quick Start
 
+In order to access driver-did-trustbloc image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
```
Comment (peacekeeper) [inline README.md:13]: Is this required for pulling the image from Github? Could you host it on DockerHub instead, to avoid the need for an access token?

Comment (fqutishat) [reply README.md]: @peacekeeper thank you for the comment. We pushed the image to DockerHub and I removed this section from README.md

**Comment by fqutishat** [comment]
@peacekeeper it's ready for review

**Comment by fqutishat** [comment]
@peacekeeper Is there any changes I can do to merge this PR?

File: docker-compose.yml [hunk 2]
```diff
@@ -107,6 +107,16 @@ services:
     image: arcblock/driver-did-abt
     ports:
       - "8101:8080"
+  driver-did-trustbloc:
+    container_name: driver-did-trustbloc
+    image: trustbloc/driver-did-trustbloc:0.1.3
+    environment:
+      - DID_METHOD_HOST_URL=0.0.0.0:8102
```
Comment (peacekeeper) [inline docker-compose.yml:114]: @fqutishat If possible, can we also use environment variables here, and set the defaults in the .env file, as the other drivers do it? If you prefer we can also do this for you (@azuzi can help)..

Comment (azuzi) [reply docker-compose.yml:114]: After merge I will do that

Comment (fqutishat) [reply docker-compose.yml:114]: @azuzi thank you so much

**Pull request merged**